data_types/softlayer_load_balancer.go

```diff
@@ -0,0 +1,25 @@
+package data_types
+
+type SoftLayer_Load_Balancer_Update_Parameters struct {
+	Parameters []SoftLayer_Load_Balancer_Update `json:"parameters"`
+}
+
+type SoftLayer_Load_Balancer_Update struct {
+	SecurityCertificateId *int `json:"securityCertificateId"`
+}
+
+type SoftLayer_Load_Balancer struct {
```

Comment:
@riethm that's a good point. Are you suggesting we simply change the filename? Or all of the associated data types as well?